es_gui/apps/data_manager/widgets.py

```diff
@@ -456,21 +456,348 @@ def execute_download(self):
 
             self.n_active_threads = len(job_batches)
 
+            five_minute_start = datetime.datetime(2017, 12, 1)
+            
             # (Re)set the progress bar and output log.
             self.progress_bar.value = 0
-            self.progress_bar.max = total_months*total_nodes + total_months
+            self.progress_bar.max = 2*total_months
```

Comment:
I'm not sure about this change, it should still be a function of the number of nodes requested. The progress bar didn't seem to match either when I tested downloading.